ui/app/services/secret-mount-path.ts

```diff
@@ -10,10 +10,8 @@ import Service from '@ember/service';
 // are not accessible
 export default class SecretMountPath extends Service {
   currentPath = '';
-  update(path) {
+
+  update(path: string) {
     this.currentPath = path;
   }
-  get() {
-    return this.currentPath;
-  }
```

Comment:
After converting to ts there was a type checking error, likely because of a conflict with the native getters. The vast majority of usages of this service were accessing `currentPath` directly so I removed the `get` method.